definitions/json/CaseEvent/CaseEvent-WA.json

```diff
@@ -75,6 +75,34 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N",
+    "SignificantEvent": "Yes",
+    "TTLIncrement": "365",
+    "Publish": "Y"
```

Comment:
Also need this on ET1Repped - Submit Draft Case

Reply:
added.